README.md

```diff
@@ -1657,3 +1658,83 @@ const Person = ({
   // ...
 };
 ```
+## 12. Google Tag Manager
+- 12.1 [What is Google Tag Manager](#12.1-what-is-google-tag-manager)
+- 12.2 [Installations](#12.2-installations)
+- 12.3 [Tags GTM](#12.3-tags-gtm)
+- 12.4 [What are triggers and what types of GTM triggers](#12.4-What-are-triggers-and-what-types-of-gtm-triggers)
+- 12.5 [What are variables inside GTM](#12.5-what-are-variables-inside-gtm)
+- 12.6 [linked tools](#12.6-linked-tools)
+- 12.7 [Test tag](#12.7-Test-tag)
+- 12.8 [Using data-gtm](#12.7-using-data-gtm)
+- 12.9 [Adding data-gtm attribute in component](#12.8-adding-data-gtm-attribute-in-component)
+### 12.1 What is Google Tag Manager
+It is a free installation tool, which simplifies tag integration and concentrates all tags on a single board. Essential for tracking site-wide metrics.
+### 12.2 Installations
+1. Go to the Google Tag Manager page at https://tagmanager.google.com/.
+2. Create a Google Tag Manager account. If you already have a Google account, just sign in.
+3. Create our first container.
+4. Copy the GTM code and add the container code to your website
+Paste this code as high as possible in the `<head>` tag of the page
+```ts
+<!-- Google Tag Manager -->
+<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+})(window,document,'script','dataLayer','GTM-PGRSV4S');</script>
+<!-- End Google Tag Manager -->
+```
+Also, paste this code after the opening `<body>` tag
+```ts
+<!-- Google Tag Manager (noscript) -->
+<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-PGRSV4S"
```

Comment:
(question): this `GTM-PGRSV4S` is our account id?

Reply:
@JulioCVaz  I removed the installation [ffb02f3](https://github.com/juntossomosmais/frontend-guideline/pull/41/commits/ffb02f341eadbc9da4095cee2155c1777cca8069)